packages/react-components/src/components/Dialpad/Dialpad.tsx

```diff
@@ -176,20 +184,33 @@ const DtmfTones: DtmfTone[] = [
 /* @conditional-compile-remove(dialpad) */
 const DialpadContainer = (props: {
   placeholderText: string;
+  deleteButtonAriaLabel: string;
```

Comment:
is this a button that we want to give a custom aria-label to? is this because you are giving a function to allow contoso to give their own behavior?

Reply:
It's a delete button so we need to have an aria-label to announce what it does

Reply:
so what I am wondering here is should we be exporting this as a prop for the `DialpadContainer` since these strings are overridable from the localization provider? I'm not sure that this is something that should be exposed as a prop.

Reply:
Also dialpad container is not exported

Reply:
Updated dialpad container props to be strings:dialpadstrings instead